packages/hooks-react/src/useForm.ts

```diff
@@ -48,11 +51,12 @@ export default function useForm<T extends GenericFormValues>({
   const [values, setValues] = useState<T>(initialValues);
   const [submitting, setSubmitting] = useState(false);
   const [errors, setErrors] = useState<FormErrors<T>>({});
+  const [validationSchemaError, setValidationSchemaError] = useState(false);
 
   const reset = useCallback(() => {
     setValues(initialValues);
     setErrors({});
-    setSubmitting(false);
+    setValidationSchemaError(false), setSubmitting(false);
```

Comment:
This is valid syntax, but not common in source code 😅 

```suggestion
    setValidationSchemaError(false); 
    setSubmitting(false);
```

Reply:
Whoops. I will fix it